docs/configuration/configuration.md

```diff
@@ -1467,6 +1467,7 @@ For OVHcloud's [public cloud instances](https://www.ovhcloud.com/en/public-cloud
 * `__meta_ovhcloud_dedicated_server_ipv6`: the IPv6 of the server
 * `__meta_ovhcloud_dedicated_server_link_speed`: the link speed of the server
 * `__meta_ovhcloud_dedicated_server_name`: the name of the server
+* `__meta_ovhcloud_dedicated_server_no_intervention`: the [intervention](https://support.us.ovhcloud.com/hc/en-us/articles/27991435200147-FAQ-Interventions-and-Hardware-Replacement) of the server.
```

Comment:
Maybe we should precise whether `no_intervention` means:
- interventions are disabled on the server.
OR
- there is no interventions currently on the server.

Reply:
Thank you for the review and sorry for my late reply, I was out of the office yesterday.

According to OVH's [document]( https://eu.api.ovh.com/console-preview/?section=%2Fdedicated%2Fserver&branch=v1#get-/dedicated/server/-serviceName-), `noIntervation` refer to:
- Prevent datacenter intervention. 

So in this case, it means "interventions are disabled on the server". And I'd like to suggest using the following description:
- `is this server immune to datacenter intervention`.

WDYT @machine424